### PR TITLE
Catalog of my things 1.0

### PR DESCRIPTION
**Pull request by gersonhiggins**
Merging Develop branch into Main
Features

1. Application to storage items:
- Storage Books,  Music albums and Games.
- Genre, Author and Label Propieties
- Data Stored on JSON files

2. Data SQL for tables
3. Video [Features](https://drive.google.com/file/d/14TUGxGBCScfQDcnYn-FwcO3FWXRBCIYm/view?usp=sharing)
4. Readme File and License

### CODE REVIEW

File: classes/item.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'securerandom'
+require_relative 'label'
+
+class Item
```
Comment (BregornOriginal) [inline classes/item.rb:4]: Excellent work implementing almost all the requirements! 🥇 

- [ ] Kindly add a test for your Item class and all these methods as it's requested on requirements.

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+CREATE TABLE games (
+    id INTEGER PRIMARY KEY,
+    multiplayer BOOLEAN NOT NULL,
+    last_played_at DATE NOT NULL,
+    item_id INTEGER NOT NULL,
+    author_id INTEGER NOT NULL,
+    FOREIGN KEY (item_id) REFERENCES items(id),
+    FOREIGN KEY (author_id) REFERENCES authors(id)
+);
```
Comment (BregornOriginal) [inline schema.sql:65]: - [ ] Kindly add indexes for each foreign key in the tables to improve the response to your queries 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
### Hi @Team 💯💯💯 

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

giphy (5)

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Gambit142 — APPROVED** [review]
Hi @gersonhiggins @enmanuelbayen @StevenWafeek 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Set up linters properly ✔️ 
- Used `Rspec` Testing Library ✔️ 
- Wrote unit tests for classes and their methods ✔️ 
- Added indexes to foreign keys ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**